confidential-data-hub/hub/Cargo.toml

```diff
@@ -69,6 +70,9 @@ sev = ["image/sev", "kms/sev", "secret/sev"]
 # support eHSM stacks (KMS, ...)
 ehsm = ["image/ehsm", "secret/ehsm"]
 
+# suppot for image-pull API
+image-pull = ["image-rs/kata-cc-rustls-tls"]
```

Comment:
With this feature, CDH is ~ 17.53MB (x86-64 & gnu). Without the feature, is ~ 16.16MB (x86-64 & gnu)

Thus I think we might not need this extra flag. We can always use `image-rs` as a dep as CDH.

Reply:
Deleted the feature